src/main/java/nextstep/optional/Users.java

```diff
@@ -13,11 +14,10 @@
             new User("honux", 45));
 
     User getUser(String name) {
-        for (User user : users) {
-            if (user.matchName(name)) {
-                return user;
-            }
-        }
-        return DEFAULT_USER;
+        Optional<User> maybeUser = users.stream()
+                .filter(user -> user.matchName(name))
+                .findFirst();
+
+        return maybeUser.orElse(DEFAULT_USER);
```

Comment:
변수 선언 없이 바로 리턴하는건 어떨까요? 

```suggestion
        return users.stream()
                .filter(user -> user.matchName(name))
                .findFirst()
                .orElse(DEFAULT_USER);
```